.github/workflows/ci.yml

```diff
@@ -73,20 +73,6 @@ jobs:
         run: |
           V=1 bash scripts/build_p2pd.sh p2pdCache 124530a3
 
-      - name: Restore deps from cache
```

Comment:
Can't we cache the deps in the same way?

Reply:
Yes, we can - given the fact that we are using lock files then we can just cache the `~/.nimble/` dir unconditionally.

Reply:
Ok nice (we should still use the hash of the lockfile has a key to avoid using the wrong cache though)

Reply:
There is no need to do that - it doesn't matter what you have in initially ~/.nimble/pkgs it will always either download the correct version of the package version or it will reuse it if it is already present.

Reply:
Sure, the nice thing about using the hash as key is that you can have multiple lockfiles deps cached in parallel, instead of having a single `pkgs` folder which may or may not be the correct one